redis/reply.hh

```diff
@@ -60,8 +60,18 @@ public:
         m->append_static(":1\r\n");
         return make_ready_future<redis_message>(m);
     }
-    static future<redis_message> zero() {
-        return err();
+    static future<redis_message> nil() {
+        auto m = make_lw_shared<scattered_message<char>> ();
+        m->append_static("$-1\r\n");
+        return make_ready_future<redis_message>(m);
+    }
+    static future<redis_message> err() {
+        return zero();
+    }
+    static future<redis_message> number(size_t n) {
```

Comment:
How about calling number() in zero(), and one()?

Reply:
@amoskong this pull request seems to conflict with your one (which changed err/zero). It's confusing....